src/CarbonAware.WebApi/src/README.md

```diff
@@ -1,7 +1,7 @@
-The Carbon Aware SDK provides an API to get the marginal carbon intensity for a given location and time period. The values reported in the Green Software Foundation's specification for marginal carbon intensity (Grams per Kilowatt Hour). In order to use the API, the environment needs to be prepared with a set of configuration parameters. Instructions on setting up the environment could be found here - [GettingStarted.md](../../../GettingStarted.md)
-
 # Carbon Aware REST API
+The Carbon Aware SDK provides an API to get the marginal carbon intensity for a given location and time period. The values reported in the Green Software Foundation's specification for marginal carbon intensity (Grams per Kilowatt Hour). In order to use the API, the environment needs to be prepared with a set of configuration parameters. Instructions on setting up the environment could be found here - https://github.com/microsoft/carbon-aware-sdk/blob/dev/GettingStarted.md
```

Comment:
Just noticed that the URL is absolute and only points to dev branch, and is not formatted and is simply a raw link.  Please update to previous format with relative link to document.

Reply:
I resolved the other conflicts, but just sending this one back for visibilty/process.

Reply:
Hi @vaughanknight, fixed GettingStarted.md link.